src/SpectatorView.Native/SpectatorView.Compositor/UnityPlugin/UnityCompositorInterface.cpp

```diff
@@ -343,6 +343,16 @@ UNITYDLL int GetVideoRecordingFrameHeight(VideoRecordingFrameLayout frameLayout)
     }
 }
 
+UNITYDLL bool IsFrameProviderSupported(int providerId)
+{
+	if (ci == nullptr)
+	{
+		ci = new CompositorInterface();
```

Comment:
How do we handle clean up of this heap allocation?

Reply:
FWIW, The unity editor never unloads plugins so the clean up occurs when the program tears down. I'll see if I can find some sort of plugin unload call for this dll, but I'm not sure if it exists.

Reply:
looks like we have  unityPluginUnload call though which seems reasonable for cleanup

Reply:
I've added a delete to the unload call.